README.md

```diff
@@ -1,2 +1,239 @@
-# math-magicians-app
-"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a randomly generated math-related quote.
+# Math-Magician App: Ometman
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="./ometman-logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+  <h3><b>Math Magicians Application: by Ometman</b></h3>
+  <h5>"Math magicians" is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to make simple calculations and read a randomly generated math-related quote.
+
+  The website has 3 pages: the home, calculator, and quotes. It consumes a public API to display math quotes. It was setup with 'create-react-app' and employs React components, props and Routing.
+  Enjoy it!
+  
+</div>
+
+# 📗 Table of Contents
+- [Math-Magician App: Ometman](#math-magician-app-ometman)
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Math Magician Project ](#-math-magician-project-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Author ](#-author-)
+    - [Authors](#authors)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+
+# 📖 Math Magician Project <a name="about-project"></a>
+
+This is just a simple SPA (Single Page Application) app that uses React framework to display contents from a home page component, calculator component and a math quote componentm which consumens an API.
+
+**A Demo Video: Introduction to this Project**
+--coming soon
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Great work on having a detailed readme file but I see that you have a title and description twice `(One before the Table of Contents and One after the Table of Contents)` in your readme file 🤔 It would be nice if you merge both of them and put them after the Table of Contents 👍 to make your readme consistent with your Table of Contents 😊

Reply:
Makes sense. Thanks.